.gitignore

```diff
@@ -50,3 +50,8 @@ bower.json
 # Ignore node_modules
 node_modules/
 
+.idea/*
+.DS_Store
```

Comment:
Place environment-specific things to your local gitignore configuration.
Some people don't use idea, for example.